internal/checker/checker.go

```diff
@@ -183,6 +182,19 @@ func (c *Checker) LoadIndex(ctx context.Context) (hints []error, errs []error) {
 		errs = append(errs, err)
 	}
 
+	c.masterIndex.MergeFinalIndexes()
```

Comment:
Merging the index is necessary to prevent duplicate index entries from messing up the calculation?

Reply:
Exactly. As the `supersedes` is not used so far (and maybe will never be used in `check`), merging the index still allows a correct computation of pack sizes. I'll add a comment here.